arviz/tests/conftest.py

```diff
@@ -7,6 +7,7 @@
 
 _log = logging.getLogger(__name__)
 
+np.random.seed(0)
```

Comment:
This should actually be in a fixture so that it runs before every test. something like

```python
@pytest.fixture(autouse=True)
def random_seed():
    np.random.seed(0)
```

Reply:
OK, will make that change!